assets/search/components/ContentSearchResults.jsx

```diff
@@ -125,7 +125,7 @@ class ContentSearchResultsComponent extends React.Component {
                             {!this.props.showTotalLabel ? null : (
                                 <React.Fragment>
                                     <span className="search-results-info__text flex-column">
-                                        <span>{gettext('search results for:')}</span>
+                                        <span>{gettext('search results for: ')}{this.props.searchParams.query}</span>
```

Comment:
might be better to have it like:
```gettext('search results for: {{query}}', {query: this.propssearchParams.query})```
or to keep that space outside gettext so it won't be missed when translating